plus/unit-testing.rst

```diff
@@ -45,24 +45,24 @@ Test Types
 Desktop
 ~~~~~~~
 
-PlatformIO wraps test and main program (from :ref:`projectconf_pio_src_dir`)
-with own |PIOUTF|, builds final program using :ref:`platform_native` and run
-test on a host machine (desktop).
+PlatformIO wraps the test and main programs (from :ref:`projectconf_pio_src_dir`)
```

Comment:
Should be "program", not "programs". The only one program will be compiled from `src` folder :)